css/style.css

```diff
@@ -1,19 +1,207 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+}
+h1 {
+    text-align: center;
+    font-weight: 5;
+    font-size: 1.5rem;
+}
+h2 {
+    font-weight: 5;
+    font-size: 1.25rem;
+    text-align: center;
+}
+/* header */
+header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 1rem 0 1rem;
+    height: 5%;
+
+}
+/* company-logo */
+.logo-area {
+    width: 25%;
+    padding: 0.5rem;
+    
+}
+.logo {
+    width: 1.5rem;
+    height: 1.5rem;
+}
+
+/* navigation */
+/* show nav and hide hamburger in large viewport */
+nav {
+    display: block
+}
+.hamburger {
+    display: none
+}
+
+/* list inside navigation */
+ul {
+    display: flex;
+    width: 90%;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-around;
+    padding-right: 10rem
+}
+
+li {
+    list-style: none;
+    cursor: pointer;
+}
+li:hover {
+    color: orange
+}
+
+
+/* main section includes main image background and info*/
+main {
+    height: 80%;
```

Comment:
This is not something we want to do as limiting the height of main means that you limit how large it can be vertically, which can cause problems when changing the heights of its contents